docs/data-collection.md

```diff
@@ -97,6 +97,9 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // subscription_sync_total is the number of times an OLM operator
   // Subscription has been synced, labelled by name and installed csv
   '{__name__="subscription_sync_total"}',
+  // csv_sync_total is the number of times an OLM operator
+  // CSV has been synced, labelled by name, version, phase, and reason
+  '{__name__="csv_sync_total"}',
```

Comment:
So the total is the current number of times in a row the version/phase/reason has been seen?

Ie if I am (“foo”, “1.0”, “deployed”, “”) 10, and suddenly it fails, will it be (“foo”, “1.0”, “failed”, “reason”) 1?

The description needs to be a bit better.

Reply:
IIRC, it's the current number of syncs for the csv by name, labelled with the other properites 

so your example would be:
 - (“foo”, “1.0”, “deployed”, “”) 10
 - (“foo”, “1.0”, “failed”, “reason”) 1